perception/object_merger/launch/object_association_merger.launch.xml

```diff
@@ -11,6 +11,6 @@
     <remap from="output/object" to="$(var output/object)"/>
     <param from="$(var data_association_matrix_path)"/>
     <param name="precision_threshold_to_judge_overlapped" value="0.4"/>
-    <param name="distance_threshold" value="9.0"/>
+    <param name="distance_threshold_list" value="9.0, 9.0, 9.0, 9.0, 9.0, 9.0, 9.0, 9.0" value-sep=", "/>
```

Comment:
Can you add description?
```
<param name="distance_threshold_list" value="9.0, 9.0, 9.0, 9.0, 9.0, 9.0, 9.0, 9.0" value-sep=", " description="UNKNOWN, CAR, TRUCK, foo... "/>
```

Reply:
Add description for class corresponding to each number in https://github.com/autowarefoundation/autoware.universe/pull/1902/commits/1bddcde845419318db542442f56e0c860643d952